lib/ui/signin/sign_in_screen.dart

```diff
@@ -0,0 +1,103 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:survey_flutter_ic/extension/context_extension.dart';
+import 'package:survey_flutter_ic/gen/assets.gen.dart';
+import 'package:survey_flutter_ic/theme/dimens.dart';
+import 'package:survey_flutter_ic/widget/flat_button_text.dart';
+import 'package:survey_flutter_ic/widget/text_input_field.dart';
+
+class SignInScreen extends StatefulWidget {
+  const SignInScreen({Key? key}) : super(key: key);
+
+  @override
+  State<SignInScreen> createState() => _SignInScreenState();
+}
+
+class _SignInScreenState extends State<SignInScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Container(
+          decoration: BoxDecoration(
+            image: DecorationImage(
+                image: AssetImage(Assets.images.bgSplash.path),
+                opacity: 0.6,
+                fit: BoxFit.fill),
+          ),
+          child: BackdropFilter(
+            filter: ImageFilter.blur(sigmaX: 10.0, sigmaY: 10.0),
+            child: Container(
+                decoration: const BoxDecoration(
+                    gradient: LinearGradient(
+                        begin: Alignment.topCenter,
+                        end: Alignment.bottomCenter,
+                        colors: [Colors.black26, Colors.black])),
+                child: _buildSignInForm()),
+          )),
+    );
+  }
+
+  Widget _buildSignInForm() {
+    return CustomScrollView(
+      slivers: [
+        SliverFillRemaining(
+          hasScrollBody: false,
+          child: Padding(
+            padding: const EdgeInsets.symmetric(
+                horizontal: space16, vertical: space10),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Expanded(
+                  child: Assets.images.icLogo
+                      .svg(width: 168, height: 40, fit: BoxFit.none),
+                ),
+                _buildSignInContent(),
+                Expanded(
+                  child: Container(),
+                ),
+              ],
+            ),
+          ),
+        )
+      ],
+    );
+  }
+
+  Widget _buildSignInContent() {
```

Comment:
@doannimble Same thought, but I think we don't reuse this form somewhere else so that's why I keep it inside the screen class, how do you think? cc @Wadeewee

Reply:
@hoangnguyen92dn In Flutter, it is recommended to split out the widget into another class whenever possible, not only for reuseable like Android Kotlin 😄 
https://docs.flutter.dev/perf/best-practices

Reply:
@doannimble Thanks for spotting out. I created a new class for the SignIn form which contains 3 input fields.